src/access/AccessControl.cpp

```diff
@@ -632,5 +632,10 @@ void SetAccessControl(AccessControl & accessControl)
     globalAccessControl = &accessControl;
 }
 
+void ResetAccessControl()
```

Comment:
Should we name to contain some `to default` part for clarity? Like `ResetAccessControlToDefault` or `SetDefaultAccessControl` ?

Reply:
Yeah, that seems useful.  Will do.

Reply:
Done.